entity-service/src/main/kotlin/com/egm/stellio/entity/repository/Neo4jRepository.kt

```diff
@@ -393,7 +393,7 @@ class Neo4jRepository(
         propertyName: String,
         datasetId: URI? = null,
         deleteAll: Boolean = false
-    ): Int {
+    ): List<URI?> {
```

Comment:
I don't get why all those hacks to return something you seem to have as a method parameter ...

Reply:
it is useful in case of deleteAll true, we need to know which datasetIds were deleted.

Reply:
why do you care of the datasetId when you delete all the instances?